src/services/AudiusBackend.js

```diff
@@ -817,7 +817,18 @@ class AudiusBackend {
     }
   }
 
-  static async upgradeToCreator(newCreatorNodeEndpoint) {
+  /**
+   * Updates a user's is_creator metadata field to true
+   */
+  static async updateIsCreatorFlagToTrue () {
+    audiusLibs.User.updateIsCreatorFlagToTrue()
```

Comment:
what's this? i dont think we should include this -- we should just stop relying on is_creator or maybe set it implicitly in libs. introducing the client dependency on this function seems debty